recipes/jsbeautifier/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "jsbeautifier" %}
+{% set version = "1.14.0" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/jsbeautifier-{{ version }}.tar.gz
+  sha256: 84fdb008d8af89619269a6aca702288b48f837a99427a0f529aa57ecfb36ee3c
+
+build:
+  number: 0
+  noarch: python
+  entry_points:
+    - js-beautify = jsbeautifier:main
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - pytest
+    - python >=3.6
+  run:
+    - editorconfig >=0.12.2
```

Comment:
This will cause a race condition in the feedstock creation and you'll have to restar the CI. If you are OK with that we can merge, if not I recommend sending one recipe at a time.

Reply:
Yes, please merge. I will restart the CI for the jsbeautifier package if it fails. Will send one recipe at a time in the future.

Reply:
Merging, thanks @mitchellsuter and @ocefpaf